Chp9Lsn11/UserInputHandler/UserInput.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UserInputHandler
+{
+    public class UserInput
+    {
+        static void Main(string[] args)
```

Comment:
I think it is pretty cool that you made a solution that contained three projects (two for code and one for testing) instead of just two (one for code and one for testing).  It is important to start down that path.  Just be sure that you adhere to:

 * "Separation of Concerns"
 * "High Cohesion"

This means that each project you create has a specific purpose.  If one of the projects you make contains user-input code, then perhaps *all* user input code should go into that project.